src/components/Overlay/Overlay.stories.mdx

```diff
@@ -0,0 +1,30 @@
+import { Meta, Story, Preview, Props } from '@storybook/addon-docs';
+import { withKnobs, boolean, array, select, text } from '@storybook/addon-knobs';
+import Overlay from './Overlay';
+import OverlayShowcase from '../storyUtils/OverlayShowcase';
+
+<Meta title="Design System/Overlay" component={Overlay} />
+
+# Overlay
+
+A universal Overlay component.
+
+# Usage
+
+```js
+import { Overlay } from '@orfium/ictinus';
+
+<Overlay opem={true} onClose={() => {}}/>;
+```
+
+# Props
+
+<Props of={Overlay} />
+
+# Overlay
```

Comment:
Add a second story "Overlay Playground" and add knobs in order for the design team to see all the variations.